src/funding/venmo/config.jsx

```diff
@@ -26,13 +26,6 @@ export function getVenmoConfig(): FundingSourceConfig {
         return false;
       }
 
-      /**
-       * Shipping callbacks will not work with Venmo unless venmo web is enabled.
-       */
-      if (!experiment?.venmoWebEnabled && shippingChange) {
```

Comment:
Shipping callback is supported in native flow through Venmo Web experience. Since it's no longer gated, we don't need this check as it will be checked in smart payment buttons to determine flow eligibility.